bin/ghe-backup

```diff
@@ -197,8 +197,18 @@ ghe-backup-audit-log || failures="$failures audit-log"
 echo "Backing up hookshot logs ..."
 ghe-backup-es-hookshot || failures="$failures hookshot"
 
+# if we should use gitbackups to backup repositories
+should_use_gitbackups_for_repositories(){
+  # TODO: Confirm re-using this setting is OK vs introducing a new setting
+  ghe-ssh "$GHE_HOSTNAME" ghe-config --true "app.github.gitbackups"
+}
+
 echo "Backing up Git repositories ..."
-ghe-backup-repositories || failures="$failures repositories"
+if should_use_gitbackups_for_repositories; then
+  ghe-backup-repositories-gitbackups || failures="$failures repositories"
```

Comment:
I am not familiar with bach script, I am wondering if we need to make `ghe-backup-repositories-gitbackups` return different exit code in order to make the failure case work?

Reply:
correct. if we can do that, then this will plug into the existing framework for error handling, which has repercussions for marking the backup as complete, as well as logging the steps that failed.